specification/maps/data-plane/Creator/preview/2022-09-01-preview/wayfind.json

```diff
@@ -0,0 +1,308 @@
+      "name": "SAS Token",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "AADToken": [
+        "https://atlas.microsoft.com/.default"
+      ]
+    },
+    {
+      "SharedKey": []
+    },
+    {
+      "SasToken": []
+    }
+  ],
+  "definitions": {
+    "Point": {
+      "description": "A point within the facility.",
+      "type": "object",
+      "properties": {
+        "latitude": {
+          "description": "the point latitude",
+          "type": "number"
+        },
+        "longitude": {
+          "description": "the point longitude",
+          "type": "number"
+        }
+      },
+      "required": [
+        "latitude",
+        "longitude"
+      ]
+    },
+    "Leg": {
+      "description": "A section of the overall path.",
+      "type": "object",
+      "properties": {
+        "mode": {
+          "type": "string",
+          "description": "The leg mode of traversal for this leg. It can be default (all on the same level), elevators or stairs when moving vertically between levels."
+        },
+        "lengthInMeters": {
+          "type": "number",
+          "description": "The leg length, in meters, of this leg."
+        },
+        "timeInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The leg travel time, in seconds, to travel between the first and last point of this leg."
+        },
+        "startLevel": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The floor where this leg starts."
+        },
+        "endLevel": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The floor where this leg ends."
+        },
+        "points": {
+          "description": "The leg shape points. Their vertical position is determined as follows: all the points are at the same height when startLevel and endLevel are the same, otherwise the array will contain only two points: the first at startLevel and the second at endLevel.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Point"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "Path": {
+      "description": "The details about a particular path between the origin and destination. It can be made up of one or more legs.",
+      "type": "object",
+      "properties": {
+        "lengthInMeters": {
+          "type": "number",
```

Comment:
For "number", you might want to specify format=double or float.

Reply:
validation won't allow it. integer and number are the only numeric schema types allowed.